utils/mil_tools/nodes/label_me_bag_example.yaml

```diff
@@ -0,0 +1,13 @@
+bags:                                       # List of bags to generate images for labeling from
+  - file: orange_pipes.bag                  # File to pull images from
+    labeled_file: orange_pipes_labeled.bag  # Output file for original data + label topics
+    segments:                               # List of named segments in bag to get images from
+      - name: path_marker_1                 # Name of segment to appear in LabelMe
+        start: 1.0                          # Duration in seconds to start getting images, optional
+        stop: 2.0                           # Duration in seconds to stop getting images, optional
+        topic: /down/left/image_raw         # Topic to pull images from
```

Comment:
It explains in the comment on the next line that if no freq is specified, every image from this time frame will be inserted into LabelMe